src/main/scala/mesosphere/marathon/core/instance/Instance.scala

```diff
@@ -418,4 +420,9 @@ object Instance {
       Json.toJson(stringToTask)
     }
   )
+
+  /**
+    * This is the default role that mesos uses when no role is specified by marathon.
+    */
+  val defaultMesosRole = "TBD"
```

Comment:
Why is this required?

Reply:
Left over from earlier code. Replaced it with "*" in the tests, wasn't used anywhere else